### PR TITLE
update: added otel fix

### PR DESCRIPTION
**Pull request by heemankv**

### CODE REVIEW

File: crates/orchestrator/src/telemetry.rs [hunk 1]
```diff
@@ -53,11 +53,21 @@ pub fn setup_analytics() -> Option<SdkMeterProvider> {
     }
 }
 
+fn get_otel_config() -> Option<OTELConfig> {
+    let otel_endpoint = get_env_var_optional("OTEL_COLLECTOR_ENDPOINT").expect("Failed to get OTEL_COLLECTOR_ENDPOINT");
+    let otel_service_name = get_env_var_optional("OTEL_SERVICE_NAME").expect("Failed to get OTEL_SERVICE_NAME");
```
Comment (apoorvsadana) [inline crates/orchestrator/src/telemetry.rs:58]: `expect` here will panic if its None

File: crates/orchestrator/src/telemetry.rs [hunk 2]
```diff
@@ -53,11 +53,21 @@ pub fn setup_analytics() -> Option<SdkMeterProvider> {
...
+
+    match (otel_endpoint, otel_service_name) {
+        (Some(endpoint), Some(service_name)) => Some(OTELConfig { endpoint, service_name }),
+        _ => None,
```
Comment (apoorvsadana) [inline crates/orchestrator/src/telemetry.rs:62]: maybe warn that one was give and the other wasn't so skipping?

File: crates/orchestrator/src/telemetry.rs (hunk 1 above)
Comment (heemankv) [reply crates/orchestrator/src/telemetry.rs:58]: `get_env_var_optional` returns a `Result<Option<String>, VarError>`

File: crates/orchestrator/src/telemetry.rs (hunk 2 above)
Comment (heemankv) [reply crates/orchestrator/src/telemetry.rs]: updated to : 
```rust
 match (otel_endpoint, otel_service_name) {
        (Some(endpoint), Some(service_name)) => Some(OTELConfig { endpoint, service_name }),
        _ => {
            tracing::warn!("OTEL_COLLECTOR_ENDPOINT or OTEL_SERVICE_NAME is not set");
            None
        }
    }
 ```

File: .env.test [hunk 3]
```diff
@@ -73,7 +73,7 @@ L1_CORE_CONTRACT_ADDRESS="0xE2Bb56ee936fd6433DC0F6e7e3b8365C906AA057"
 
 ##### SNOS #####
 ## This is needed right now because Madara doesn't support getProof
-RPC_FOR_SNOS=""
+RPC_FOR_SNOS="http://81.16.176.130:9545"
```
Comment (ocdbytes) [inline .env.test:76]: we need to remove this can't push this URL.

File: crates/orchestrator/src/workers/update_state.rs [hunk 4]
```diff
@@ -11,15 +11,20 @@ use crate::workers::Worker;
...
 pub struct UpdateStateWorker;
 
+/// Worker that creates a new state transition job for the next set of blocks to be settled.
 #[async_trait]
 impl Worker for UpdateStateWorker {
     async fn run_worker(&self, config: Arc<Config>) -> color_eyre::Result<()> {
         tracing::trace!(log_type = "starting", category = "UpdateStateWorker", "UpdateStateWorker started.");
+        // A single single state transition job can process a contiguous set of blocks.
 
+        // gets latest state transition job that was registered in the database
```
Comment (ocdbytes) [inline crates/orchestrator/src/workers/update_state.rs:21]: few of the comments here are not needed. Ig variable names are self explanatory.

Comment (heemankv) [reply crates/orchestrator/src/workers/update_state.rs]: removed.

File: .env.test (hunk 3 above)
Comment (heemankv) [reply .env.test]: removed.

**Review by apoorvsadana — APPROVED** [review]

**Review by ocdbytes — APPROVED** [review]

**Pull request merged**